exampleCourse/questions/templateFixedMultipleChoiceManyAnswers/question.html

```diff
@@ -0,0 +1,20 @@
+<pl-question-panel>
+What color can be found in a rainbow?
+</pl-question-panel>
```

Comment:
```suggestion
<pl-question-panel>
<p>What color can be found in a rainbow?</p>
<p><i>This question will randomly select one of the correct answers and as many of the incorrect answers as needed to make up the full list, and will then randomly shuffle them.</i></p>
</pl-question-panel>
```

Do you think we should include brief descriptions like this?

Reply:
I'd omit the description... One less thing to have to delete.